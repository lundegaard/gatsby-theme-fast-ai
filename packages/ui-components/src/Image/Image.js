import React from 'react';
import { Image as RebassImage } from 'rebass';

const Image = props => <RebassImage {...props} />;

export default Image;
