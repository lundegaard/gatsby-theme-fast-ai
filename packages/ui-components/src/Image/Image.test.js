import React from 'react';
import { shallow } from 'enzyme';

import Image from './Image';

describe('<Image />', () => {
	const requiredProps = {};

	it('matches snapshot when shallowly rendered', () => {
		const wrapper = shallow(<Image {...requiredProps} />);
		expect(wrapper).toMatchSnapshot();
	});
});
