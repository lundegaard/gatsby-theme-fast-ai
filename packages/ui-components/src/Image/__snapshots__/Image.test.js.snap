// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Image /> matches snapshot when shallowly rendered 1`] = `<ForwardRef />`;
